--- src/uimac/MyController.m.orig	2021-06-23 16:30:30.000000000 +0200
+++ src/uimac/MyController.m	2021-09-08 11:58:06.000000000 +0200
@@ -186,7 +186,7 @@
           (![[NSFileManager defaultManager]
               /* BCP 6/2016: Changed from /usr/bin/unison for El Capitan, per
                  suggestion from Alan Shutko */
-                 fileExistsAtPath:@"/usr/local/bin/unison"]) )
+                 fileExistsAtPath:@"__MACPORTS_PREFIX__/bin/unison"]) )
           [self raiseCltoolWindow:nil];
 }
 
